config/file_list.json

```diff
@@ -34,6 +34,31 @@
         "data18_13TeV.periodO.physics_Main.PhysCont.DAOD_STDM3.grp23_v01_p4433",
         "data18_13TeV.periodQ.physics_Main.PhysCont.DAOD_STDM3.grp23_v01_p4433"
     ],
+    "Signal_WWZ":[
+        "mc16_13TeV.364243.Sherpa_222_NNPDF30NNLO_WWZ_4l2v_EW6.deriv.DAOD_STDM3.e5887_s3126_r9364_p4434",
+        "mc16_13TeV.364243.Sherpa_222_NNPDF30NNLO_WWZ_4l2v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10201_p4434",
+        "mc16_13TeV.364243.Sherpa_222_NNPDF30NNLO_WWZ_4l2v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10724_p4434",
+        "mc16_13TeV.364244.Sherpa_222_NNPDF30NNLO_WWZ_2l4v_EW6.deriv.DAOD_STDM3.e5887_s3126_r9364_p4434",
+        "mc16_13TeV.364244.Sherpa_222_NNPDF30NNLO_WWZ_2l4v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10201_p4434",
+        "mc16_13TeV.364244.Sherpa_222_NNPDF30NNLO_WWZ_2l4v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10724_p4434",
+        "mc16_13TeV.363507.Sherpa_222_NNPDF30NNLO_WWZ_3l1v2j_EW6.deriv.DAOD_STDM3.e6379_s3126_r9364_p4434 ",
+        "mc16_13TeV.363507.Sherpa_222_NNPDF30NNLO_WWZ_3l1v2j_EW6.deriv.DAOD_STDM3.e6379_s3126_r10201_p4434",
+        "mc16_13TeV.363507.Sherpa_222_NNPDF30NNLO_WWZ_3l1v2j_EW6.deriv.DAOD_STDM3.e6379_s3126_r10724_p4434"
+    ],
+    "Signal_WZZ":[
+        "mc16_13TeV.364245.Sherpa_222_NNPDF30NNLO_WZZ_5l1v_EW6.deriv.DAOD_STDM3.e5887_s3126_r9364_p4434",
+        "mc16_13TeV.364245.Sherpa_222_NNPDF30NNLO_WZZ_5l1v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10201_p4434",
+        "mc16_13TeV.364245.Sherpa_222_NNPDF30NNLO_WZZ_5l1v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10724_p4434",
+        "mc16_13TeV.364246.Sherpa_222_NNPDF30NNLO_WZZ_3l3v_EW6.deriv.DAOD_STDM3.e5887_s3126_r9364_p4434",
+        "mc16_13TeV.364246.Sherpa_222_NNPDF30NNLO_WZZ_3l3v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10201_p4434",
+        "mc16_13TeV.364246.Sherpa_222_NNPDF30NNLO_WZZ_3l3v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10724_p4434",
+        "mc16_13TeV.363508.Sherpa_222_NNPDF30NNLO_WZZ_4l2j_EW6.deriv.DAOD_STDM3.e6374_s3126_r9364_p4434",
+        "mc16_13TeV.363508.Sherpa_222_NNPDF30NNLO_WZZ_4l2j_EW6.deriv.DAOD_STDM3.e6374_s3126_r10201_p4434",
+        "mc16_13TeV.363508.Sherpa_222_NNPDF30NNLO_WZZ_4l2j_EW6.deriv.DAOD_STDM3.e6374_s3126_r10724_p4434",
+        "mc16_13TeV.363509.Sherpa_222_NNPDF30NNLO_WZZ_3l1v2j_EW6.deriv.DAOD_STDM3.e6374_s3126_r9364_p4434",
+        "mc16_13TeV.363509.Sherpa_222_NNPDF30NNLO_WZZ_3l1v2j_EW6.deriv.DAOD_STDM3.e6374_s3126_r10201_p4434",
+        "mc16_13TeV.363509.Sherpa_222_NNPDF30NNLO_WZZ_3l1v2j_EW6.deriv.DAOD_STDM3.e6374_s3126_r10724_p4434"
+    ],
```

Comment:
Should we add the ZZZ contribution as well? 
In the nominal results with v2.3, ZZZ contributed with 4.1 events (out of a total of 30.1 signal on-shell triboson events)

```suggestion
    ],
    
    "Signal_ZZZ":[
        "mc16_13TeV.364247.Sherpa_222_NNPDF30NNLO_ZZZ_6l0v_EW6.deriv.DAOD_STDM3.e5887_s3126_r9364_p4434",
        "mc16_13TeV.364247.Sherpa_222_NNPDF30NNLO_ZZZ_6l0v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10201_p4434",
        "mc16_13TeV.364247.Sherpa_222_NNPDF30NNLO_ZZZ_6l0v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10724_p4434",
        "mc16_13TeV.364248.Sherpa_222_NNPDF30NNLO_ZZZ_4l2v_EW6.deriv.DAOD_STDM3.e5887_s3126_r9364_p4434",
        "mc16_13TeV.364248.Sherpa_222_NNPDF30NNLO_ZZZ_4l2v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10201_p4434",
        "mc16_13TeV.364248.Sherpa_222_NNPDF30NNLO_ZZZ_4l2v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10724_p4434",
        "mc16_13TeV.364249.Sherpa_222_NNPDF30NNLO_ZZZ_2l4v_EW6.deriv.DAOD_STDM3.e5887_s3126_r9364_p4434",
        "mc16_13TeV.364249.Sherpa_222_NNPDF30NNLO_ZZZ_2l4v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10201_p4434",
        "mc16_13TeV.364249.Sherpa_222_NNPDF30NNLO_ZZZ_2l4v_EW6.deriv.DAOD_STDM3.e5887_s3126_r10724_p4434",
        "mc16_13TeV.367730.Sherpa_222_NNPDF30NNLO_ZZZ_4l2j_EW6.deriv.DAOD_STDM3.e8416_s3126_r10201_p4434",
        "mc16_13TeV.367730.Sherpa_222_NNPDF30NNLO_ZZZ_4l2j_EW6.deriv.DAOD_STDM3.e8416_s3126_r10724_p4434",
        "mc16_13TeV.367730.Sherpa_222_NNPDF30NNLO_ZZZ_4l2j_EW6.deriv.DAOD_STDM3.e8416_s3126_r9364_p4434"
    ],
```